code_quality.js

```diff
@@ -0,0 +1,211 @@
+const gitChangedFiles = require('git-changed-files');
+const prettier = require('prettier');
+const prettierConfig = require('./prettier_config.json');
+const codeQualityConfig = require('./package.json').codeQuality || {};
+
+const fs = require('fs');
+const { ESLint } = require('eslint');
+const eslint = new ESLint({ fix: true });
+
+const getAllFiles = dir => {
+   let results = [];
+   const list = fs.readdirSync(dir);
+   list.forEach(function (file) {
+      file = dir + '/' + file;
+      const stat = fs.statSync(file);
+      if (
+         file.endsWith('node_modules') ||
+         file.indexOf('.git') !== -1 ||
+         file.indexOf('/dist/') !== -1 ||
+         file.indexOf('\\dist\\') !== -1
```

Comment:
Apparently there's a decent number of files which also shouldn't be included in this
- package-lock.json
Minified bundle files:
- *.min.js 
- *.min.css
- *.bundle.js
-  and some other similiar ones
 We can do this easier with a regexp in a one liner instead, i.e.:
`if (/.git|.min.css|.min.js/gi.test(file))` 
You'd have to extend and [test](https://regex101.com/) that regexp I'm not 100% sure.

Reply:
Add function:

const fileToExclude = (file, stat) => {
   if (stat && stat.isDirectory()) {
      return file.endsWith('node_modules') ||
        file.indexOf('.git') !== -1 ||
        file.indexOf('/dist/') !== -1 ||
        file.indexOf('\\dist\\') !== -1;
   }
   const fileName = path.basename(file);
   return (/(.git|.min.css|.min.js)$/.test(fileName));
};